tests/testDipoleFitter.py

```diff
@@ -55,17 +56,25 @@ class DipoleFitTestGlobalParams(object):
     @var offsets: pixel coordinates between lobes of dipoles
     """
 
-    def __init__(self, xc=np.array([65.3, 24.2]), yc=np.array([38.6, 78.5])):
-        """Initialize the parameters."""
+    def __init__(self, xc=None, yc=None):
+        """Initialize the parameters.
```

Comment:
This sentence isn't a helpful docstring. I would actually have no docstring (except the parameters), since the class level docstring is the descriptive one.